res/qml/MixxxSlider.qml

```diff
@@ -0,0 +1,35 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import Mixxx 1.0
+
+Item {
+    id: root
+    property alias group: control.group
+    property alias key: control.key
+
+    Slider {
+        id: slider
+        orientation: Qt.Vertical
+        value: control.parameter
+        wheelEnabled: true
+        width: parent.width
+        height: parent.height
+
+        Control {
+            id: control
+            parameter: slider.value
+        }
+
+        handle: Image {
+            source: "../skins/LateNight/classic/sliders/knob_volume_deck.svg"
```

Comment:
Use generically named variables for the paths to the graphics assets ("slider-background", "slider-handle"). https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/QML/near/136221715